src/library/DIPS.Mobile.UI/Components/Loading/StateView/StateView.Properties.cs

```diff
@@ -56,65 +54,62 @@ public bool ShouldFadeBetweenStates
     /// <summary>
     /// The view model to configure <see cref="StateView"/>
```

Comment:
Legg til kommentar om at denne MÅ lages av konsumer :) Husk også å endre wiki for dette slik at det er lett å komme i gang.